up42/order.py

```diff
@@ -13,6 +14,20 @@
 LIMIT = 200
 
 
+class OrderParams(TypedDict):
+    """
+    Represents the stucture data format for the order parameters.
+    dataProduct: The dataProduct id for the specific product configuration.
+    params: Order parameters for each product. \
+        They are different from product to product depending on product schema.
+    tags: User tags to helping to identify the order.
+    """
+
+    dataProduct: str
+    params: Dict[str, Any]
```

Comment:
We know at least that this at least contains id and aoi

Reply:
id yes, aoi no always, I think for the full-scene products they do not have aoi.